deployment/helm/human-connection/README.md

```diff
@@ -0,0 +1,72 @@
+# Helm installation of Human Connection
+
+Deploying Human Connection with Helm is very straight forward. All you have to
+do is to change certain parameters, like domain names and API keys, then you
+just install our provided Helm chart to your cluster.
+
+## Configuration
+
+You can customize the network with your configuration by changing the `values.yaml`, all variables will be available as
+environment variables in your deployed kubernetes pods.
+
+Probably you want to change this environment variable to your actual domain:
+
+```bash
+# in folder /deployment/helm
+CLIENT_URI: "https://develop.human-connection.org"
+```
+
+If you want to edit secrets, you have to `base64` encode them. See [kubernetes documentation](https://kubernetes.io/docs/concepts/configuration/secret/#creating-a-secret-manually). You can also use `helm-secrets`, but we have yet to test it.
+
+```bash
+# example how to base64 a string:
+$ echo -n 'admin' | base64
+YWRtaW4=
+```
+Those secrets get `base64` decoded and are available as environment variables in
+your deployed kubernetes pods.
+
+# https
+If you start with setting up the `https`, when you install the app, it will automatically take care of the certificates for you.
+
+First check that you are using `Helm v3`, this is important since it removes the need for `Tiller`. See, [FAQ](https://helm.sh/docs/faq/#removal-of-tiller)
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2020-02-04T13:02:21Z" title="Tuesday, February 4th 2020, 2:02:21 pm +01:00">Feb 4, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -0,0 +1,72 @@
+# Helm installation of Human Connection
+
+Deploying Human Connection with Helm is very straight forward. All you have to
+do is to change certain parameters, like domain names and API keys, then you
+just install our provided Helm chart to your cluster.
+
+## Configuration
+
+You can customize the network with your configuration by changing the `values.yaml`, all variables will be available as
+environment variables in your deployed kubernetes pods.
+
+Probably you want to change this environment variable to your actual domain:
+
+```bash
+# in folder /deployment/helm
+CLIENT_URI: "https://develop.human-connection.org"
+```
+
+If you want to edit secrets, you have to `base64` encode them. See [kubernetes documentation](https://kubernetes.io/docs/concepts/configuration/secret/#creating-a-secret-manually). You can also use `helm-secrets`, but we have yet to test it.
+
+```bash
+# example how to base64 a string:
+$ echo -n 'admin' | base64
+YWRtaW4=
+```
+Those secrets get `base64` decoded and are available as environment variables in
+your deployed kubernetes pods.
+
+# https
+If you start with setting up the `https`, when you install the app, it will automatically take care of the certificates for you.
+
+First check that you are using `Helm v3`, this is important since it removes the need for `Tiller`. See, [FAQ](https://helm.sh/docs/faq/#removal-of-tiller)
```


Good to know! :+1: